src/screens/HomePage/HomepageStructuredData.tsx

```diff
@@ -3,17 +3,26 @@ import { getMapImageUrl } from 'common/urls';
 import { Helmet } from 'react-helmet';
 
 const HomepageStructuredData: React.FC = () => {
-  const jsonData = JSON.stringify({
+  const imageData = JSON.stringify({
     '@context': 'https://schema.org',
     '@type': 'ImageObject',
     url: getMapImageUrl(),
     width: 1200,
     height: 630,
   });
+  const organizationData = JSON.stringify({
+    '@context': 'https://schema.org',
+    '@type': 'Organization',
+    name: 'Covid Act Now',
+    url: 'https://covidactnow.org/',
+    logo:
+      'https://covidactnow.org/static/media/covid-act-now-logo.0ac0983b.png',
+  });
```

Comment:
The logo URL changes every time the original asset is changed - we need to import the logo and use the imported variable as URL:

```tsx
import canLogoUrl from 'assets/images/can-logo.png';

// ...
const organizationData = JSON.stringify({
  // ...
  logo: canLogoUrl
})
```

Reply:
ahh got it.